### ISSUE TITLE
[TODO] Init App

### ISSUE THREAD

**Issue by mkelly08** [guismicios/django_chaman#3]
# TODO (Describe what needs to be done and how it should be done)
We need to create a basic structure for the project. 

It has to have the following:

- Create a Setup.py 
- Set up the folder structure.
- Create requirements.txt for development.
- We should use pipenv and create the first pipfile and pipfile.lock

### More Information: (Here put extra context for the todo. It should also contain the minimum requirements so it can be closed.)

The objective of this issue is to create the basic structure of the python project. where we can send it to pypi and publish it. 

## Time: (Time to do this issue)
1 hr

**Comment by SantiagoTorres** [comment]
I wonder if we'd like to take a look at projects such as [skaffold](https://skaffold.dev/) and [draft](https://github.com/Azure/draft) for inspiration. I also wonder if we'd like to generalize this tool for other technologies (e.g., pick an orchestration system, pick your CI infra, etc.)

**Comment by mkelly08** [comment]
Yes, that's exactly what we are looking for but with docker. Yes, we are looking on making a generalized pick your favorite technology. But I think we should start by doing something simple for the main users (AKA Us) And from there we should be adding more and more features.

The next time we talk IRL we will explain to you the full project so we can take care of some issues:

* The project structure.
* The architecture of the project.
* First milestones
* And what should be the first release.
